doc/specifications/rest/REST-API-V2.rst

```diff
@@ -111,7 +121,33 @@ In this document, for the sake of readability, no prefix is used in the URIs. In
 prefixes all REST hrefs.
 
 Remember that URIs to REST resources should never be generated manually, but obtained from earlier REST
- calls.
+calls.
+
+Alias URIs
+----------
+
+By default entity URIs are constructed using a (server local) URL template and
+the database ID of the referenced entity. While this mechanism might be subject
+to change (e.g. to support multi-server/-domain setups with interlinking)
+users are requesting more intuitive ways to identify entities to reference
+entities while creating/updating related entities. Alias URIs help to achieve
+this by providing more human readable resource identifiers. These URIs only
+support GET/HEAD operations and will return a redirect to the cannonical entity
+URL. However, when using them as a reference in a POST/PUT/PATCH request, the
+server will accept them and automatically reference the correct entity.
+
+A request/response cycle for an alias URI is supposed to be executed as follows:
+
+:Resource: /content/types/_by_identifier/article
```

Comment:
Any reason why you aim to duplicate the existing mechanism for this?
Referring here to `?remoteId`, `?identifier`, `?locationPath`, `?roleId`, `?email` and `?login`, I would have expected the concepts here to extend that for the identified scenarios.

For instance the example here can already be expressed using:
```
<contentTypeByIdentifier media-type="" href="/api/ezp/v2/content/types{?identifier}"/>
```